### ISSUE TITLE
Test suite can't compile

### ISSUE THREAD

**Issue by mdorman** [k0ral/atom-conduit#2]
I'm guessing this is because of https://prime.haskell.org/wiki/Libraries/Proposals/MonadFail:
```
test/Main.hs:218:5: error:
    • No instance for (Control.Monad.Fail.MonadFail Gen)
        arising from a do statement
        with the failable pattern ‘Just content’
    • In a stmt of a 'do' block:
        Just content <- fromNullable . pack <$> listOf1 alphaNum
      In the expression:
        do Just content <- fromNullable . pack <$> listOf1 alphaNum
           AtomGenerator <$> arbitrary <*> arbitrary <*> pure content
      In an equation for ‘arbitrary’:
          arbitrary
            = do Just content <- fromNullable . pack <$> listOf1 alphaNum
                 AtomGenerator <$> arbitrary <*> arbitrary <*> pure content
    |
218 |     Just content <- fromNullable . pack <$> listOf1 alphaNum
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by k0ral** [comment]
Fixed in 6689393fe8b1984781a94bd514e54e6fe29e9a4c.
Released as [0.5.0.3](https://hackage.haskell.org/package/atom-conduit-0.5.0.3)

Thank you for reporting.

**Comment by mdorman** [comment]
Thanks for such a quick fix!
